servicetest/testframework/lib.py

```diff
@@ -118,50 +118,51 @@ def start(self, data):
 
             The values in the dict are passed as arguments to the D-Bus methods on SoftwareContainerAgent.
             The dict is mapped like so:
-            Container.CONFIG - argument to SoftwareContainerAgent::CreateContainer
-            Container.HOST_PATH - second argument to SoftwareContainerAgent::BindMountFolderInContainer
-            Container.BIND_MOUNT_DIR - third argument to SoftwareContainerAgent::BindMountFolderInContainer
-            Container.READONLY - fourth argument to SoftwareContainerAgent::BindMountFolderInContainer
+            Container.CONFIG - argument to SoftwareContainerAgent::Create
+            Container.HOST_PATH - second argument to SoftwareContainerAgent::BindMount
+            Container.BIND_MOUNT_DIR - third argument to SoftwareContainerAgent::BindMount
+            Container.READONLY - fourth argument to SoftwareContainerAgent::BindMount
         """
         success = self.__create_container(data[Container.CONFIG])
         if False == success:
             print "Failed to create container"
             return False
         
-        self.__bind_dir, success = self.__bindmount_folder_in_container(data[Container.HOST_PATH],
-                                                               data[Container.BIND_MOUNT_DIR],
-                                                               data[Container.READONLY])
+        success = self.__bindmount_folder_in_container(data[Container.HOST_PATH],
+                                                       data[Container.BIND_MOUNT_DIR],
+                                                       data[Container.READONLY])
         if False == success:
```

Comment:
I think it's more typical to Python to check this like "if success is False". There's no risk of assignment then either and it doesn't have to be set up backwards looking.